lib/smart_answer_flows/inherits-someone-dies-without-will.rb

```diff
@@ -111,10 +111,6 @@ def define
         save_input_as :parents
 
         on_condition(variable_matches(:region, 'england-and-wales')) do
-          on_condition(variable_matches(:partner, 'yes')) do
-            next_node_if(:outcome_21, responded_with('yes'))
-            next_node_if(:siblings?, responded_with('no'))
-          end
           on_condition(variable_matches(:partner, 'no')) do
```

Comment:
I'm a bit confused. I actually think we probably can remove the use of `on_condition`, but leave the inner `next_node_if`'s, because the only way to get to this section of code is by responding 'no' to `:partner`.

That would mean that this:

```
on_condition(variable_matches(:partner, 'no')) do
  next_node_if(:outcome_3, responded_with('yes'))
  next_node_if(:siblings?, responded_with('no'))
end
```

would become:

```
next_node_if(:outcome_3, responded_with('yes'))
next_node_if(:siblings?, responded_with('no'))
```

Is that what you meant, @tadast?

@floehopper: Do you see a problem with removing these out `on_condition`'s?